docs/source/getting_started/10_minutes_to_graviti.rst

```diff
@@ -4,3 +4,183 @@
 #######################
  10 Minutes to Graviti
 #######################
+
+This is a simple introductory tutorial for beginners.
+
+**************
+ Registration
+**************
+
+Before using Graviti SDK, please finish the following registration steps:
+
+- Please visit `Graviti`_ to sign up.
+- Please visit `Graviti Developer Tools`_ to get an AccessKey.
+
+.. _Graviti: https://gas.graviti.com/tensorbay/
```

Comment:
why the first url has a trailing slash, but second does not?

Reply:
I'm not sure what the reason is, but this is the url of current website

Reply:
ref:
https://searchfacts.com/url-trailing-slash/